### PR TITLE
NXP-28512: make RenderDocument and RenderDocumentFeed use the method input

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

**Review by kevinleturc — APPROVED** [review]

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java [hunk 1]
```diff
@@ -66,8 +66,8 @@
     protected String charset = "UTF-8";
 
     @OperationMethod
-    public Blob run(DocumentModelList docs) throws OperationException, RenderingException, TemplateException,
-            IOException {
+    public Blob run(DocumentModelList docs)
+            throws OperationException, RenderingException, TemplateException, IOException {
```
Comment (efge) [inline modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java:70]: I'm pretty sure that Eclipse doesn't indent like that

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocument.java [hunk 2]
```diff
@@ -65,7 +65,8 @@
 
     @OperationMethod(collector = BlobCollector.class)
     public Blob run(DocumentModel doc) throws OperationException, RenderingException, TemplateException, IOException {
-        String content = RenderingService.getInstance().render(type, template, ctx);
+        OperationContext subCtx = ctx.getSubContext(true, doc);
```
Comment (efge) [inline modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocument.java:68]: I don't know this code well but all the uses of `getSubContext` I can find all use try-with-resources to close it when done.

**Review by efge — CHANGES_REQUESTED** [review]

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java (hunk 1 above)
Comment (ataillefer) [reply modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java:70]: It does :smile:

**Review by ataillefer — APPROVED** [review]

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java (hunk 1 above)
Comment (troger) [reply modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java:70]: I'm using the Eclipse formatter with the Nuxeo Eclipse config so...

**Review by ataillefer — APPROVED** [review]

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java (hunk 1 above)
Comment (efge) [reply modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java:70]: Yes indeed, my mistake. Must be something new in it...

**Review by efge — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**